### ISSUE TITLE
Extract mapping of RN Accessibility to UIA from DynamicAutomationPeer

### ISSUE THREAD

**Issue by SavoySchuler** [microsoft/react-native-windows#3111]
# Proposal: Extract mapping of RN Accessibility to UIA from DynamicAutomationPeer

## Summary

Right now, DAP is the only control that reads out the RN accessibility properties we set and translates them into UIA properties. Views are the only things that use it. We would have to subclass all of our native controls to use DAP which is bad for perf. 

Need to develop new story for custom automation peers for WinUI 3.0.

**Comment by chrisglein** [comment]
I believe this is the WinUI improvement that this is waiting on: https://github.com/microsoft/microsoft-ui-xaml/issues/1349

**Comment by chrisglein** [comment]
Not something we plan to address on the Paper renderer as we move towards Fabric.
Fabric will not use XAML for the fundamentals of accessibility here, so this isn't applicable.
